admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -138,22 +182,33 @@
         type: Object,
         required: true,
       },
+      emptyImage: {
```

Comment:
`emptyImageUrl` would have been more clear, but since it's a String I guess it's ok